homeassistant/components/goalzero/binary_sensor.py

```diff
@@ -28,9 +28,17 @@ async def async_setup_entry(hass, entry, async_add_entities):
 class YetiBinarySensor(YetiEntity, BinarySensorEntity):
     """Representation of a Goal Zero Yeti sensor."""
 
-    def __init__(self, api, coordinator, name, sensor_name, server_unique_id):
+    def __init__(
+        self,
+        api,
```

Comment:
Please type the signature completely if typing any parameter.

Reply:
Are we good here now?